web/params.go

```diff
@@ -363,3 +357,25 @@ func ParamsFromRequest(r *http.Request) *Params {
 
 	return params
 }
+
+// getPerPageFromQuery returns the PerPage value from the given query.
+// This function should be removed and the support for `pageSize`
+// should be dropped after v1.46 of the mobile app is no longer supported
+func getPerPageFromQuery(query url.Values) int {
+	val, err := strconv.Atoi(query.Get("per_page"))
+	if err != nil {
+		val, err = strconv.Atoi(query.Get("pageSize"))
+		if err != nil || val < 0 {
+			return PerPageDefault
+		} else if val > PerPageMaximum {
+			return PerPageMaximum
+		}
+		return val
```

Comment:
I think we can remove this and rely on the check below. Just need to check for the error as well.